solanaj/src/main/java/org/p2p/solanaj/model/types/Encoding.kt

```diff
@@ -0,0 +1,14 @@
+package org.p2p.solanaj.model.types
+
+import com.google.gson.annotations.SerializedName
+
+enum class Encoding(
```

Comment:
Encoding for what?
RequestEncoding?
RpcParamsEncoding?

Reply:
This encoding is for everything

Reply:
This encoding is for everything

Reply:
This encoding is for everything.
Again, let's make it simple and short